src/qibo/quantum_info/metrics.py

```diff
@@ -385,3 +385,92 @@ def gate_error(channel, target=None, backend=None):
     error = 1 - average_gate_fidelity(channel, target, backend=backend)
 
     return error
+
+
+def meyer_wallach(circuit, backend=None):
```

Comment:
I think `meyer_wallach_entanglement` is a better (even though longer) name, because it's clearer